explore/src/main/scala/explore/config/ObsConfigurationPanel.scala

```diff
@@ -94,25 +96,28 @@ object ObsConfigurationPanel:
         def posAngleEditor(pa: View[Angle]) =
           <.div(
             ExploreStyles.InputWithLabel,
-            InputWithUnits(
+            FormInputTextView(
               id = "pos-angle-value".refined,
-              clazz = Css.Empty,
+              groupClass = ExploreStyles.PAConfigurationAngle,
               value = pa,
-              units = "° E of N",
+              postAddons = List("° E of N"),
               disabled = !props.agsState.get.canRecalculate,
               validFormat = MathValidators.truncatedAngleDegrees,
               changeAuditor = ChangeAuditor.bigDecimal(3.refined, 2.refined)
             )
           )
 
-        Form(size = Small)(
-          ExploreStyles.Compact,
-          ExploreStyles.ObsConfigurationForm
+        <.div(
+          LucumaStyles.FormColumnCompact,
+          ExploreStyles.PAConfigurationForm
         )(
-          <.label("Position Angle", HelpIcon("configuration/positionangle.md".refined)),
-          EnumViewSelect(
+          FormLabel(htmlFor = "pos-angle-alternative".refined)(
+            "Position Angle",
+            HelpIcon("configuration/positionangle.md".refined)
+          ),
+          EnumDropdownView(
             clazz = ExploreStyles.ObsConfigurationObsPA,
-            id = "pos-angle-alternative",
+            id = "pos-angle-alternative".refined,
             value = posAngleOptionsView,
             disabled = !props.agsState.get.canRecalculate
           ),
```

Comment:
FYI. You could use a FormEnumDropdownView here to combine the label and the dropdown. IIRC, it does require using a fragment for the label when you have the help icon, but you don't have to repeat the id/htmlFor.

Reply:
I tried this but I'm not sure how to add an (i) button there, Any ideas?